test/integration/ae_mdw_web/controllers/activity_controller_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule Integration.AeMdwWeb.ActivityControllerTest do
```

Comment:
`BlockchainSim` was a nice module tool created to easily allow adding new transactions to a blockchain from scratch. It could be used here to be able to have automated unit/controller tests. Even only with `:spend_tx`s, could help validating the activities pagination (if continuation is correct) and if all are returned (later could be used for validating the filters).

Reply:
Validating all of these future use cases can be done using integration tests, so I would avoid too much mocking that would make the tests lose their value

Reply:
The reason for the proposal is because there were many data that I needed to test manually. It could be automated somehow to avoid manual regression tests before releases. Besides that we can't have proper integration tests with `backward` direction. This is what the current tests would be missing for both directions:

- if all activities are from the account (on directions)
- if the activities pages are complete in a way that the key boundary doesn't exclude some entry (for both directions)
- if the next continues from the last of previous page without repetition
- if the endpoint renders all fields and if these fields have proper value

Reply:
I've added some unit tests now